game/requesthandler.cc

```diff
@@ -1,376 +1,235 @@
 #include "requesthandler.hh"
 #include "unicode.hh"
 #include "game.hh"
-
-#ifdef USE_WEBSERVER
```

Comment:
do we want to include the webserver within the core functionality? If yes, then this is the way to go. If no then we still have to support it as a 'module' being captured within `USE_WEBSERVER`